app/views/payments/new.html.haml

```diff
@@ -0,0 +1,18 @@
+.container
+  .row
+    .col-xs-6.col-xs-offset-3
+      %h1
+        Payment Summary : 
+        = humanized_money_with_symbol @total_amount_to_pay
+      - if @payment.errors.any?
+        .alert.alert-danger
+          %ul
+            - @payment.errors.full_messages.each do |msg|
+              %li= msg
+    .col-xs-8.col-xs-offset-2.well
+      = render partial: 'payment'
+  .row
+    .col-md-13
+      %p.text-muted.text-center
+        %small
+          The payment is totally secure. Your credit card details will be sent directly to our payment processor.
```

Comment:
In American English, "totally" in this context sounds colloquial or even sarcastic.
I would say: 
> All payments are handled securely by our payment processor, [Stripe](https://stripe.com/about).